app/src/main/java/com/infomaniak/swisstransfer/ui/navigation/NavigationItem.kt

```diff
@@ -39,8 +40,9 @@ enum class NavigationItem(
     @StringRes val label: Int,
     val icon: ImageVector,
     @StringRes val contentDescription: Int,
+    val destination: MainNavigation,
 ) {
-    SENT(R.string.appName, Icons.AutoMirrored.Filled.Send, R.string.appName),
-    RECEIVED(R.string.appName, Icons.Default.Star, R.string.appName),
-    SETTINGS(R.string.appName, Icons.Default.Settings, R.string.appName),
+    SENT(R.string.appName, Icons.AutoMirrored.Filled.Send, R.string.appName, SentDestination),
+    RECEIVED(R.string.appName, Icons.Default.Star, R.string.appName, ReceivedDestination),
+    SETTINGS(R.string.appName, Icons.Default.Settings, R.string.appName, SettingsDestination),
```

Comment:
I don't know what will be expected in the app in terms of accessibility. But it remains to be seen whether it's useful or not, and I've added it here to serve as a default.

Reply:
With the `Rail` there is no label

Reply:
Actually, can't we simply pass the label to the contentDescription of the Icon instead of requiring to specify a second contentDescription field inside the enum and make sure we keep the contentDescription and label values exactly the same?

Reply:
I'm going to delete it and manage it differently instead.

Reply:
[Updated] Now there's no contentDescription, just the label